specification/desktopvirtualization/resource-manager/Microsoft.DesktopVirtualization/preview/2021-05-13-preview/desktopvirtualization.json

```diff
@@ -2317,6 +2318,13 @@
             "schema": {
               "$ref": "#/definitions/SessionHostPatch"
             }
+          },
+          {
+            "name": "force",
+            "in": "query",
+            "required": false,
+            "type": "boolean",
+            "description": "Force flag to update assign unassign or reassign personal desktop."
```

Comment:
1. add comma between "assign" and "unassign".
2. Just want to bring up, should we call it as "confirmation" flag instea of "force" flag?

Reply:
i think we want to stay with force